packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -81,27 +81,28 @@ This is a starting point for browser-based OpenID Connect flows such as the impl
+| request_uri                                    | Location where the authorization request payload data is referenced in an authorization request to the `/authorize` endpoint.                                                                                                                                                                                                                                                                                                                                                                                                                      | Query       | JWT       | FALSE      |
+| request                                        | A JWT created by the client that enables requests to be passed as a single, self-contained parameter. See [Parameter details](#parameter-details).                                                                                                                                                                                                                                                                                                                                                                                                 | Query       | JWT       | FALSE      |
+| scope                                          | `openid` is required for authentication requests. Other [scopes](#access-token-scopes-and-claims) may also be included.                                                                                                                                                                                                                                                                                                                                                                                                                            | Query       | String    | TRUE       |
+| sessionToken                                   | Okta one-time session token. This allows an API-based user sign-in flow (rather than the Okta sign-in UI). Session tokens can be obtained via the [Authentication API](/docs/reference/api/authn/).                                                                                                                                                                                                                                                                                                                                                | Query       | String    | FALSE      |
```

Comment:
```suggestion
| sessionToken                                   | Okta one-time session token. This allows an API-based user sign-in flow (rather than the Okta sign-in page). Session tokens can be obtained through the [Authentication API](/docs/reference/api/authn/).                                                                                                                                                                                                                                                                                                                                                | Query       | String    | FALSE      |
```